packages/main/src/server/cache-manager.test.ts

```diff
@@ -53,6 +53,7 @@ describe("GoogleAICacheManager", () => {
       systemInstruction: "talk like a cat",
       tools: [{ functionDeclarations: [{ name: "myFn" }] }],
       toolConfig: { functionCallingConfig: {} },
+      displayName: "a display name.",
```

Comment:
If adding it to the call, you should probably add an expectation that it's been added to the body, otherwise no reason to add it here (other than to see there's no error). See the other requestBody expectations below.

Reply:
Update these tests.
  - now tests that displayName is in the request.
  - added model as a constant at the top of the test, and use the correct model name for launch. (I know it doesn't matter what the model name is, but it's good to be consistent.)